inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/resample_ref.cl

```diff
@@ -54,8 +54,13 @@ KERNEL (resample_gpu_ref)(__global INPUT0_TYPE* input,
     typedef MAKE_VECTOR_TYPE(OUTPUT_TYPE, PACK_SIZE) out_pack_t;
 
     const int ox = get_global_id(0);
+#if OUTPUT_DIMS <= 4
+    const int oy = get_global_id(1);
+    const int oz = 0;
+#else
     const int oy = get_global_id(1) % OUTPUT_SIZE_Y;
     const int oz = get_global_id(1) / OUTPUT_SIZE_Y;
+#endif
     const int feature = (get_global_id(2) * PACK_SIZE) % OUTPUT_FEATURE_NUM;
     const int batch = (get_global_id(2) * PACK_SIZE) / OUTPUT_FEATURE_NUM;
```

Comment:
Could you add conversion to int/uint for global_ids to avoid int64 math?

Reply:
Done